### PR TITLE
Db/cycle two

### PR DESCRIPTION
**Pull request by maymaays**
## What kind of change does this PR introduce?
update promotion, design, and orderinfo
## Description
- delete product as be foreign key 
- add status field
- add new events
(promotion)
- add yard in design models
(design)
- add remarks (by using customer size)
(orderinfo)
## The PR fulfills these requirements:
- [ ] merge with latest version in `dev` branch
- [ ] run pass all testcase, if it have

## Other information:
...

### CODE REVIEW

File: cart/models.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ def __str__(self):
 
 class OrderInfo(models.Model):
     quantity = models.IntegerField(default=0)
+    remarks = models.CharField(max_length=100, default=0)
```
Comment (kamontat) [inline cart/models.py]: Change remark to Textfield

File: catalog/fixtures/init_promotion.yaml [hunk 2]
```diff
@@ -1,40 +1,65 @@
 - model: catalog.promotion
```
Comment (kamontat) [inline catalog/fixtures/init_promotion.yaml:1]: Change description to real description and move out date to different column 

Maybe separate to start date and end date column

File: catalog/models.py [hunk 3]
```diff
@@ -10,6 +10,8 @@ class Design(models.Model):
     description = models.CharField(max_length=150)
     price = models.DecimalField(
         max_digits=8, decimal_places=2, default=0.00)  # max: 999,999.99
+    yard = models.DecimalField(
+        max_digits=8, decimal_places=2, default=0.00)
```
Comment (kamontat) [inline catalog/models.py]: Did 8 digit is too long for design yard?

**Review by kamontat — CHANGES_REQUESTED** [review]

**Comment by maymaays** [comment]
already edited code na ja

**Review by kamontat — APPROVED** [review]

**Pull request merged**